vars/capsuleSystemTests.groovy

```diff
@@ -117,9 +117,87 @@ void call(Map additionalConfig=[:], parametersOverride=[:]) {
             }
             steps {
               dir('system-tests/scripts') {
-                sh 'make build-binaries'
+                sh 'make check'
```

Comment:
I think we can the `check` step once before `prepare environment` parallel step. Because it is checking the following things:

```

	@if [ ! -d "${SYSTEM_TEST_DIR}" ]; then \
		echo -e "\n\tError: missing system-tests repo.\n\tPlease checkout system-tests into ${SYSTEM_TEST_DIR}.\n"; \
		exit 1; \
	fi
	@if [ ! -d "${VEGA_DIR}" ]; then \
		echo -e "\n\tError: missing vega core repo.\n\tPlease checkout vega core into ${VEGA_DIR}.\n"; \
		exit 1; \
	fi
	@if ! which poetry; then \
		echo -e "\n\tError: No poetry installed. Install it manually on your computer.\n"; \
		exit 1; \
	fi
	@if [ ! -d "${VEGACAPSULE_DIR}" ]; then \
		echo -e "\n\tError: missing vegacapsule directory. \n\tPlease checkout vegacapsule into ${VEGACAPSULE_DIR}.\n"; \
		exit 1; \
	fi;
	@if [ ! -e "${SYSTEM_TEST_DIR}/vegacapsule/capsule_config.hcl" ]; then \
		echo -e "\n\tError: The ${SYSTEM_TEST_DIR}/vegacapsule/capsule_config.hcl file is missing.\n"; \
		exit 1; \
	fi;
	@if [ -z "${GOBIN}" ]; then \
		echo -e "\n\tError: the GOBIN environment variable is not set.\n\tPlease set it to your go binaries path(usually: <home-path>/go/bin)"; \
		exit 1; \
	fi;
```


So it will be more clear to check it before, but I am also ok to leave it as it is. It's up to you :)